logic/account.js

```diff
@@ -227,7 +225,7 @@ class Account {
 			fields = null;
 		}
 
-		filter.multisig = true;
+		filter.multiMin_gt = 0;
```

Comment:
Better variable name here possible? maybe `multisigMin`?

Reply:
Its a database field so can't be changed (at the moment)